sqlalchemy_utils/types/arrow.py

```diff
@@ -1,3 +1,4 @@
+from arrow import Arrow
```

Comment:
This results in an `ImportError` if arrow isn't installed; the current architecture is to allow for imports within the `sqlalchemy_utils` package but only raise an exception if a class is used but the underlying dependency isn't installed.